utils/_decorators.py

```diff
@@ -187,8 +188,12 @@ def compute_declaration(only_for_library, component_name, declaration, tested_ve
                 return declaration
 
             # declaration must be now a version number
-            if tested_version >= declaration:
-                return None
+            if declaration.startswith("v"):
+                if tested_version >= declaration:
+                    return None
+            else:
+                if semver.Version(str(tested_version)) in semver.NpmSpec(declaration):
```

Comment:
`semver.Version()` will throw if test_version is not a valid semver, should we put a try catch ?